COMPATIBILITY.md

```diff
@@ -0,0 +1,14 @@
+# Compatibility
+
+A list of programs this is currently compatible with.
+Please note that this list is not complete.
+
+|Program|Compatiblity|
+|---|---|
+|Google Chrome|Yes|
+|Zoom|Yes|
```

Comment:
Can we add a note about re-codesigning Zoom? Maybe an asterisk that links out to the instructions in one of the issues? Or write up text to go below here.